i18n/es/acknowledgements.md

```diff
@@ -1,8 +1,9 @@
-En adición a los contribuidores de Boost y OpenSSL, y a todos los programadores desde los tiempos inmemoriales, que han hecho este proyecto posible, queremos reconocer lo siguiente:
+## Reconocimientos
+Adicional a los contribuyentes de Boost y OpenSSL, y a todos los programadores desde los tiempos inmemoriales, que han hecho este proyecto posible, queremos reconocer lo siguiente:
 
-- Wei Dai, Jeffrey Walton, y todos los desarrolladores en Cryoto++ por proveer una influencial y libre librería de criptografía en C++
-- Dean Michael Berris, Glyn Matthews, y todos los desarrolladores en cpp-netlib por proveer una librería activamente desarrollada cross-plataforma de red.
-- zzz, str4d, y todos los desarrolladores de I2P Java, en ambos, pasado y presente, que tienen un set de estándares a seguir para las demás implementaciones de I2P
-- orion por proveer ```i2pcpp```: la implementación en C++ [original](http://git.repo.i2p.xyz/w/i2pcpp.git) de I2P
-- orignal por proveer ```i2pd```: la mas completa implementación de I2P [de la cual poder copiar](https://github.com/purplei2p/i2pd/commit/45d27f8ddc43e220a9eea42de41cb67d5627a7d3)
-- EinMByte por mejorar todas las implementaciones en C++ de I2P (Incluyendo Kovri)
+- A Wei Dai, Jeffrey Walton y a todos los desarrolladores de Crypto++ por entregarnos una libreria criptográfica de C++ libre de gran influencia (o, "con una gran influencia").
```

Comment:
Please don't leave alternative translations in the final text. I'm not qualified to comment on Spanish grammar, but `de gran influencia` is a well-known phrase and ` (o, "con una gran influencia")` can probably be simply removed.